taxonomies/ingredients.txt

```diff
@@ -56199,12 +56202,12 @@ wikidata:en:Q1076906
 nutriscore_fruits_vegetables_nuts:en:no
 
 <en:sunflower seed
-en:shelled sunflower seeds
-de:Sonnenblumenkerne geschält
-fi:kuoreton auringonkukansiemen, kuorettomat auringonkukansiemenet
-fr:graines de tournesol décortiquées
-it:semi di girasole sbucciati
-pl:nasiona słonecznika łuskane, łuskane nasiona słonecznika
+# en:shelled sunflower seeds
```

Comment:
If everything is commented out, we need to remove the <en:sunflower seed on top

Reply:
<en:sunflower seed
is now commented as well (not removed, if future contributor see it commented that will be a hint to understand that they are handled using processing taxonomy. Whereas if we remove the whole block there is risk that a future contributor add it in the taxonomy (although it should be noticed during code review))

Reply:
ah ok my bad, I didn't look correctly